cache/blobs.go

```diff
@@ -486,12 +480,16 @@ func ensureCompression(ctx context.Context, ref *immutableRef, comp compression.
 	return err
 }
 
-func commitOverlaybd(ctx context.Context, sr *immutableRef, desc *ocispecs.Descriptor) error {
+func commitOverlayBD(ctx context.Context, sr *immutableRef, desc *ocispecs.Descriptor) error {
 	snStat, err := sr.cm.Snapshotter.Stat(ctx, sr.getSnapshotID())
 	if err != nil {
 		return errors.Wrapf(err, "failed to Stat overlaybd")
 	}
-	dir := path.Dir(snStat.Labels[obdlabel.LocalOverlayBDPath])
+	bdPath := snStat.Labels[obdlabel.LocalOverlayBDPath]
+	if bdPath == "" {
+		return errors.New("missing overlaybd path label")
+	}
+	dir := path.Dir(bdPath)
```

Comment:
Maybe redundant; mostly thought; perhaps we need to make sure that this code path isn't hit if there's no label

Reply:
This label is generated from `overlaybd.Prepare`, but I think it needs to be judged here too.